sci-electronics/splat/splat-1.4.2.ebuild

```diff
@@ -42,30 +44,28 @@ src_configure() {
 }
 
 src_compile() {
-	local CC=$(tc-getCC) CXX=$(tc-getCXX)
+	tc-export CXX
 
-	${CXX} -Wall ${CXXFLAGS} ${LDFLAGS} itwom3.0.cpp splat.cpp -o rfsplat -lm -lbz2 || die
+	${CXX} -Wall ${CPPFLAGS} ${CXXFLAGS} ${LDFLAGS} itwom3.0.cpp splat.cpp -o rfsplat -lm -lbz2 || die
 	if use hires; then
-		cp "${S}/hires.h" "${S}/splat.h"
-		${CXX} -Wall ${CXXFLAGS} ${LDFLAGS} itwom3.0.cpp splat.cpp -o rfsplat-hd -lm -lbz2 || die
+		cp {hires,splat}.h || die
+		${CXX} -Wall ${CPPFLAGS} ${CXXFLAGS} ${LDFLAGS} itwom3.0.cpp splat.cpp -o rfsplat-hd -lm -lbz2 || die
```

Comment:
Oh hadn't noticed those were actually multi-object.. the whole implicit rules thing works with .cpp too, e.g.
```
emake LDLIBS="-lm -lbz2" -E "splat: itwom3.0.o"
```
gives
```
x86_64-pc-linux-gnu-g++ -march=native -O2 -pipe -Wl,-O1 -Wl,--as-needed splat.cpp itwom3.0.o  -lm -lbz2 -o splat
```

Reply:
Okay I think I've got it now